lib/ansible/cli/galaxy.py

```diff
@@ -188,6 +196,9 @@ def init_parser(self):
                                  'https://galaxy.ansible.com/me/preferences.')
         common.add_argument('-c', '--ignore-certs', action='store_true', dest='ignore_certs',
                             default=C.GALAXY_IGNORE_CERTS, help='Ignore SSL certificate validation errors.')
+        common.add_argument('--timeout', dest='timeout', type=int,
```

Comment:
We could also add common arguments for username and password.

cc @newswangerd since you've requested this a couple times :smile:

Reply:
@s-hertel that would be really nice for vanilla pulp ansible installations, which don't currently have token auth.

Reply:
i'll leave that to a followup